about.html

```diff
@@ -0,0 +1,203 @@
+            <p class="email">international.conference@conference</p>
+          </div>
+        </div>
+      </section>
+    </header>
+    <!-- MAIN SECTION -->
+    <main class="main-container">
+      <div class="main-top">
+        <div class="conference-title-desc">
+          <h2>
+            International Conference on Sustainable Water Management ( ICSWM )
+          </h2>
+          <p>
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Illum,
+            exercitationem maiores quisquam ipsam provident maxime inventore
+            officiis in facere
+          </p>
+        </div>
+        <div class="conference-logo">
+          <img
+            src="./assets/Images/conference-logo.png"
+            alt="conference-logo"
+          />
+        </div>
+      </div>
+      <div class="main-bottom">
+        <div class="main-bottom-title">
+          <h3>Our Past Conference Keynote</h3>
+          <div class="underline"></div>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti
+            architecto quibusdam tempore voluptatibus dolor et aliquam amet
+            animi aspernatur quasi.
+          </p>
+        </div>
+        <div class="past-submit-container">
+          <div class="past-submit">
+            <div class="text-content">
+              <h2>2022</h2>
+              <p>
+                International Conference on Sustainable Water Management
+                (ICSWM-2022)
+              </p>
+              <div class="color-overlay"></div>
+            </div>
+          </div>
+          <div class="past-submit">
+            <div class="text-content">
+              <h2>2022</h2>
+              <p>
+                International Conference on Sustainable Water Management
+                (ICSWM-2022)
+              </p>
+              <div class="color-overlay"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <section class="partner-section">
+        <div class="partner-section-title">
+          <h2>Partners</h2>
+          <div class="underline"></div>
+        </div>
+        <div class="partner-section-container">
+          <ul class="partners">
+            <li class="partner">
+              <img src="./assets/Images/associate-2.jpg" alt="" />
+            </li>
+          </ul>
+        </div>
+      </section>
+    </main>
+    <!-- FOOTER SECTION -->
+    <footer class="footer">
+      <div class="footer-container">
+        <div class="footer-logo">
+          <img
+            src="./assets/Images/conference-logo.png"
+            alt="conference-logo"
+          />
+        </div>
+        <div class="footer-content">
+          <p>Copyright © 2022, All Rights Reserved By Conferencefora</p>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Veniam quis
+            soluta sint dolorem ad rerum magnam
+          </p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly change the colors of the footer of the desktop version of your 'about page'. In the template, you follow the color of this section is inverted and it has a dark background with white text.
 - Template:
 - 
<img width="766" alt="image" src="https://user-images.githubusercontent.com/61919617/203629516-e6197ebd-956d-4134-b1c1-5c4a4d325f23.png">